lib/src/over_react_test/zone_util.dart

```diff
@@ -44,3 +45,11 @@ void zonedExpect(actual, matcher, {String reason}) {
     expect(actual, matcher, reason: reason);
   });
 }
+
+/// Sets the zone that React components are run in.
+///
+/// By default, tests and React run in differing zones. This can be used to force
+/// tests and components to be run in the same zone.
+void setComponentZone([Zone zone]) {
```

Comment:
Since this is essentially the public API we're adding, I think it would be nice to add an example to the doc comment _(especially since we don't really have any other form of over_react_test documentation)_.